packages/@o3r/application/schemas/cms.schema.json

```diff
@@ -29,11 +29,43 @@
     },
     "functionalContentsFolder": {
       "type": "string",
-      "description": "The relative path to the functional contents metadata folder to import"
+      "description": "The relative path to the functional contents metadata folder to import",
+      "default": "functional-contents-metadata"
+    },
+    "agateEnabled": {
+      "type": "boolean",
+      "description": "Specify if Agate is enabled or not in the preview",
+      "default": false
+    },
+    "baseNpmName": {
+      "type": "string",
+      "description": "if agateEnabled is set to true, this is used to set the proper Agate form type in the preview"
+    },
+    "features": {
+      "type": "object",
+      "description": "Set of additional CMS features that can be activated or deactivated for the application",
+      "properties": {
+        "themingCssOverrideEnabled": {
+          "type": "boolean",
+          "description": "Enable CSS override customization in CMS theming administration",
```

Comment:
Better with "Specifies that the blueprint supports theming CSS file customization"